supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -29,7 +29,9 @@ In the *Name* field, enter `test-postgresql`.
 [[user-interface-elements]]
 == User interface elements
 
-Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable.
+Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
+
+If an element is not labeled in the GUI, refer to the element by a generic description and do not use bold text. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the *Search* field".
```

Comment:
I wonder if adding a note would be useful (I know the PatternFly link has already been provided) to inform a doc user that if they do not know the name of a tooltip item then they should refer to the Components section of the PatternFly docs? The kebab icon comes to mind, but luckily this is not under the *Components* section list, but the term "kebab" is embedded in the Dropdown sub-section. 

https://www.patternfly.org/v4/components/overflow-menu/html
https://www.patternfly.org/v4/components/dropdown/html/#kebab

Reply:
Might be safer not to include such links? The term "kebab icon" should not be used in our docs :)

Reply:
Yes, best not to link to the "dropdown menu" link, but the general "components" link would be useful. We already mention PatternFly, so one could locate the kebab menu if they go digging.

Reply:
I think I'd prefer not to link to PatternFly here, because the ISG does override some of the guidance there with the names of the elements. So it's probably the ISG that people should refer to more closely for if they don't know what the UI element is called. (And we can't link to the ISG).